languages.yaml

```
@@ -1311,6 +1311,7 @@ Jupyter:
       - "#9E9E9E"
     chip: "#DA5B0B"
   serialization: jupyter-notebooks
+  tokei_line_types: [code, comments]
```

```suggestion
  line_types: [code, comments]
```
minor: I don't think we should expose internal terminology in this file 🤔

---

:thinking: I kind of like exposing "tokei" (at least for now) so that potential contributors will know that they need to reference tokei to see the available "line types". Though perhaps it would be better to add to `CONTRIBUTING.md` instead.